css/style.css

```diff
@@ -0,0 +1,722 @@
+/* stylelint-disable no-descending-specificity */
+:root {
+  --color-primary: #272a31;
+  --color-secondary: #d3d3d3;
+  --color-tertiary: #ec5242;
+  --color-secondary-dark: rgba(73, 71, 71, 0.83);
+}
+
+@font-face {
+  font-family: 'Cocogoose-Regular';
+  src: url("fonts/Cocogoose-Pro-trial.ttf");
+}
+
+@font-face {
+  font-family: 'Cocogoose-thin';
+  src: url("fonts/Cocogoose-Pro-thin-trial.ttf");
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+  margin: 0;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  line-height: 1.6;
+  color: var(--color-primary);
+  background: var(--color-secondary);
+}
+
+a {
+  text-decoration: none !important;
+}
+
+footer {
+  padding: 1.6rem;
+  display: flex;
```

Comment:
- [ ] **Optional:** It could be nice to extract some rule that re-occur like `display: flex;`